api/v1/types_util.go

```diff
@@ -531,6 +533,15 @@ func (ispn *Infinispan) GetLogCategoriesForConfig() map[string]string {
 	return copied
 }
 
+// GetLogPatternForConfig returns the log pattern or empty string
+func (ispn *Infinispan) GetLogPatternForConfig() string {
+	var pattern = DefaultLoggingPattern
+	if ispn.Spec.Logging != nil {
```

Comment:
You also need to check that `ispn.Spec.Logging.Pattern != ""` as the current code will return an empty string if the user provides:

```yaml
  logging:
    categories:
      org.infinispan: debug
      org.jgroups: debug
```